Java/src/main/java/epam/Planes/MilitaryPlane.java

```diff
@@ -0,0 +1,45 @@
+package epam.Planes;
+
+import epam.models.MilitaryType;
+
+import java.util.Objects;
+
+
+public class MilitaryPlane extends Plane {
+
+	private MilitaryType type;
+
+	public MilitaryPlane(String model, int maxSpeed, int maxFlightDistance, int maxLoadCapacity, MilitaryType type) {
+		super(model, maxSpeed, maxFlightDistance, maxLoadCapacity);
+		this.type = type;
+	}
+
+	public MilitaryType getType() {
+		return type;
+	}
+
+	@Override
+	public String toString() {
+		return super.toString().replace("}", ", type=" + type + '}');
+	}
+
+	@Override
+	public boolean equals(Object o) {
```

Comment:
change the name of variable "o", variable names must be self descriptive

Reply:
Done